### PR TITLE
Fix typos and improve README.md format

### PR DESCRIPTION
**Pull request by daniloisr**
I also added line breaks for `long long` lines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,7 +127,9 @@ query = Product.
 
 query.to_params
 
-#=> {:filter=>{:min_price=>"30", :max_price=>"65"}, :fields=>{:product=>"name,price", :manufacturer=>"name,website"}, :include=>"manufacturer", :sort=>"-popularity,price", :page=>{:limit=>10}}
+#=> {:filter=>{:min_price=>"30", :max_price=>"65"},
+  :fields=>{:product=>"name,price", :manufacturer=>"name,website"},
```
Comment (thiagoa) [inline README.md]: Shouldn't this line start with a `#` to not confuse with actual code? And the line below as well.

File: README.md [hunk 2]
```diff
@@ -266,7 +275,9 @@ Munson.configure(url: 'http://api.example.com') do |c|
 end
 ```
 
-Each Munson::Resource has its own Munson::Client. The client *copies* the default connection so its easy to set general configuration options, and overwrite them on a resource by resource basis.
+Each Munson::Resource has its own Munson::Client. The client *copies* the
+default connection so its easy to set general configuration options, and
```
Comment (thiagoa) [inline README.md]: I guess `its` == `it's`

File: README.md [hunk 3]
```diff
@@ -292,22 +303,24 @@ Munson.configure(url: 'http://api.example.com', response_key_format: :dasherize)
...
-Two special options can be passed into ```.configure```:
-* ```url``` the base url for this endpoint
-* ```response_key_format``` the format of the JSONAPI response keys. Valid values are: ```:dasherize```, ```:camelize```, ```nil```
+Two special options can be passed into `.configure`:
+* `url` the base url for this endpoint
+* `response_key_format` the format of the JSONAPI response keys. Valid
+  values are: `:dasherize`, `:camelize`, `nil`
 
-Additinally any Faraday Connection options can be passed. [Faraday::Connection options](https://github.com/lostisland/faraday/blob/master/lib/faraday/connection.rb Faraday::Connection)
+Additinally any Faraday Connection options can be passed. [Faraday::Connection options](https://github.com/lostisland/faraday/blob/master/lib/faraday/connection.rb)
```
Comment (thiagoa) [inline README.md]: `Additinally` == `Additionally,`

File: README.md [hunk 4]
```diff
@@ -342,7 +356,9 @@ class Album
   # Register the type w/ munson
   Munson.register_type(munson.type, self)
 
-  # When you aren't inherited from Munson::Resource, Munson will pass a Munson::Document to a static method called munson_initializer for you to initialze your record as you wish
+  # When you aren't inherited from Munson::Resource, Munson will pass a
+  # Munson::Document to a static method called munson_initializer for you to
+  # initialze your record as you wish
```
Comment (thiagoa) [inline README.md]: `initialze` == `initialize`

File: README.md [hunk 5]
```diff
@@ -361,7 +377,9 @@ albums.first.title #=> An album title!
 
 ### Any ol' object (Register type, add munson_initializer)
 
-As long as a class is registered with munson and it response to munson_initializer, Munson will be able to initialize the object with or without a client
+As long as a class is registered with munson and it response to
```
Comment (thiagoa) [inline README.md]: `response` == `responds to`

File: README.md (hunk 1 above)
Comment (daniloisr) [inline README.md]: Good point.

About the line below, this is a pattern that is being used in the whole readme, we may need to discuss it and change all the occurrences, but I'm fine to proceed as it is.

Also, about the grammar errors, it wasn't my intention to fix them on this PR, but would be nice to review the whole document. I'll be fixing the ones that you pointed for now.

**Pull request merged**